models/users_model.js

```diff
@@ -0,0 +1,23 @@
+const mongoose = require("mongoose");
+const bcrypt = require("bcrypt");
+
+const userSchema = new mongoose.Schema(
+    {
+        username: {type: String, required: true, unique: true},
```

Comment:
Why the username should be unique? In every social media, I can create 2 accounts with the same username.